src/components/common/TmSessionExplore.vue

```diff
@@ -144,7 +143,7 @@ export default {
       return selectedNetwork
     },
     addressPrefixes() {
-      return this.networks.map(({ address_prefix }) => address_prefix)
+      return this.networks //.map(({ address_prefix }) => address_prefix)
```

Comment:
why? ^^

Reply:
It is working for me. I get ` ["cosmos", "cosmos", "terra", "terra", "emoney", " "]`

One question. Shouldn't Polkadot address_prefix be rather "" than " "? Why the blank space there?

Reply:
but why? this.networks is not this.addressPrefixes....

Reply:
In networkOfAddress function it's using like an array of objects, containing address_prefix and testnet property